src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.model;
+
+public class Fruit {
+    private String name;
+    private int quantity;
+
+    public Fruit(String name, int quantity) {
+        this.name = name;
+        this.quantity = quantity;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void addQuantity(int quantity) {
+        this.quantity += quantity;
+    }
+
+    public void reduceQuantity(int quantity) {
+        this.quantity -= quantity;
```

Comment:
Consider adding validation in the `reduceQuantity` method to ensure that the quantity does not become negative. This will prevent logical errors when reducing the quantity.